.changeset/slimy-walls-worry.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/data-table': minor
```

Comment:
Please be careful when to use which version types. Minor is mostly used for new features or some substantial changes. For bug fixes, minor changes, normal dep updates, etc. we should use `patch`.

```suggestion
'@commercetools-uikit/data-table': patch
```

Reply:
> FYI: if a PR closes or fixes an issue, we should use the syntax `Closes` or `Fixes`, which then instructs GitHub to automatically close the issue.
> 
> <img alt="image" width="253" src="https://user-images.githubusercontent.com/1110551/190205015-5a534cb0-9b08-4ef1-adb5-534ff9e9ead3.png">

Thank you. I get it now. Updated here:

https://github.com/commercetools/ui-kit/pull/2284/commits/13f01a6db493060ec99c82a2e33e8f846aa07727